### DISCUSSION TITLE
[Application Development]  ERROR [ui 6/6] RUN npx prisma generate --schema schema.prisma

### DISCUSSION THREAD

**Discussion by L2nitant** [ethereum-optimism/developers#115] [Application Development]
### Issue Description

I am trying to run indexer/ui on my op-stack chain with docker compose up.
chain is currently running on sepolia network (server) and blockscout on my local
.env fle
INDEXER_L1_ETH_RPC=alchemy url
INDEXER_L2_ETH_RPC=l2 url
DATABASE_URL=db URL

.toml file
[chain]
preset = $INDEXER_CHAIN_PRESET

L1 Config
l1-polling-interval = 0
l1-header-buffer-size = 0
l1-confirmation-depth = 0
l1-starting-height = 0

L2 Config
l2-polling-interval = 0
l2-header-buffer-size = 0
l2-confirmation-depth = 0

[rpcs]
l1-rpc = li RPC
l2-rpc = l2 RPC

[db]
host = "localhost"
port = 5432
user = "postgres"
password = "paswword"
name = "name"

[http]
host = "127.0.0.1"
port = 8080

[metrics]
host = "127.0.0.1"
port = 7300

After running docker compose up 
these are my logs 
```
Attaching to indexer-api-1, indexer-backend-goerli-1, indexer-indexer-1, indexer-migrations-1, indexer-postgres-1, indexer-prisma-check-1, indexer-ui-1
indexer-api-1             | t=2023-12-07T13:42:57+0000 lvl=info msg="running api..." role=api
indexer-api-1             | t=2023-12-07T13:42:57+0000 lvl=eror msg="failed to decode config file" role=api err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-api-1             | t=2023-12-07T13:42:57+0000 lvl=eror msg="failed to load config"        role=api err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-api-1             | t=2023-12-07T13:42:57+0000 lvl=eror msg="application failed"           err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-migrations-1      | t=2023-12-07T13:42:57+0000 lvl=info msg="running migrations..." role=migrations
indexer-migrations-1      | t=2023-12-07T13:42:57+0000 lvl=eror msg="failed to decode config file" role=migrations err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-migrations-1      | t=2023-12-07T13:42:57+0000 lvl=eror msg="failed to load config"        role=migrations err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-migrations-1      | t=2023-12-07T13:42:57+0000 lvl=eror msg="application failed"           err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-api-1 exited with code 1
indexer-migrations-1 exited with code 1
indexer-indexer-1         | t=2023-12-07T13:42:58+0000 lvl=info msg="running indexer..." role=indexer
indexer-indexer-1         | t=2023-12-07T13:42:58+0000 lvl=eror msg="failed to decode config file" role=indexer err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-indexer-1         | t=2023-12-07T13:42:58+0000 lvl=eror msg="failed to load config"        role=indexer err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-indexer-1         | t=2023-12-07T13:42:58+0000 lvl=eror msg="application failed"           err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
indexer-indexer-1 exited with code 1
dependency failed to start: container indexer-indexer-1 exited (1)
```

**Comment by L2nitant** [comment]
After running 
`npx prisma studio --schema schema.prisma`
on indexer/ui 
Prigma Studio is open on browser but it shows 
`The table "public.l1_bridge_deposits" does not exist in the current database.` for every table in prisma studio
I also checked it in my postgres, these table do not exist there.. there are lot of other table in postgres

**Comment by sbvegan** [comment]
@L2nitant can you help us out and clarify your question by editing your original post?

> I am trying to run indexer/ui on my op-chain with docker compose up.

- What indexer/ui?
- Are you following a tutorial or resource we've created?

**Reply by L2nitant** [reply]
https://github.com/ethereum-optimism/optimism/tree/op-node/v1.3.0/indexer/ui
I am following this

**Reply by sbvegan** [reply]
I'm going to see if I can find someone else with more domain knowledge to take a look. However, one thing I noticed in the logs you provided was:

```
msg="failed to decode config file" role=api err="toml: line 6 (last key \"chain.preset\"): expected value but found '\\n' instead"
```

This looks like that the syntax in the configuration toml file is off. There might be trailing newline characters (`\n`) that is causing an issue. Additionally, your toml file's password is set to:

```
password = "paswword"
```

This might be a typo.
